ote/src/clj/ote/services/taxiui_service.clj

```diff
@@ -121,8 +122,22 @@
 
 (defn fetch-operating-areas
   [db {filter :filter}]
+  ; TODO auth checks
   (vec (list-operating-areas db {:term (str "%" filter "%")})))
 
+(defn fetch-service-summaries
+  [db user {}]
+  [{:id "service-id-num"
+    :name "namenamename"
+    :updated "23.12.2021"
+    :example-trip 36.90
+    :price-per-kilometer 1.5
+    :price-per-minute 1
+    :operating-areas [{:label "Jokualue" :primary? true}]}]
```

Comment:
These values above seem to be some testing data and would still need to be replaced with proper values, I guess.

Reply:
whoopsie, will remove